README.md

```diff
@@ -302,7 +302,8 @@ These variables are the defaults of our roles, if you want to override the prope
 | prometheus_url                  | Prometheus server url                                                                                            | http://localhost:9090                                 |
 | web_host                        | Host where the web instance is listening                                                                         | http://localhost                                      |
 | install_method                  | Installation method for trento components, can be either `rpm` or `docker`                                       | rpm                                                   |
-
+| generate_certs | Whether the needed certificates will be generated by the script automatically | true |
```

Comment:
ocd: can we try to have the table width the same with the upper rows? The rest of the rows have the same width until now

Reply:
Done in https://github.com/trento-project/ansible/pull/57/commits/b0df2d4cf1cd533d0a870d155bf6405288550322